### DISCUSSION TITLE
UW in Plasma KDE

### DISCUSSION THREAD

**Discussion by serrq** [horizontalsystems/unstoppable-wallet-android#5965] [General]
I am planning to migrate to Plasma Mobile OS.
One of mine issues I have not a wallet there.

Might you develop UW for Manjaro KDE Plasma OS?

**Comment by 3point14159** [comment]
I'll throw my hat in for desktop binaries. All in due time, methinks.

**Comment by serrq** [comment]
I believe that it be necessary to build a team to porting UW in Plasma environment. I mean a dedicated team.

**Comment by 3point14159** [comment]
I'm happy with building a strong, feature-rich, matured mobile app first, really dominate and stand-out in this space, then I think that might attract the desktop developer talent to join-up for full platform support.

But hey, whatever happens, happens. The community is full of surprises.
